account.py

```diff
@@ -32,6 +32,8 @@ class Configuration(metaclass=PoolMeta):
         'Not allow post out invoices without AEAT SII Keys'))
     not_allow_in_invoices_aeat_sii_keys = fields.MultiValue(fields.Boolean(
         'Not allow post in invoices without AEAT SII Keys'))
+    sii_default_limit_date = fields.MultiValue(fields.Integer('SII Default Limit Date',
+        help='Default limit date for SII reports in days'))
```

Comment:
Will be better to define the field as an offset of days, not a "limit date", as the field is an integer not a date. May be like:

sii_default_offset_days = fields.MultiValue(fields.Integer('SII Default Offset Days',
    help='Default offset days in the invoices search for the SII Books'))